packages/app/src/systems/Send/hooks/useSend.tsx

```diff
@@ -338,14 +338,15 @@ export function useSend() {
   );
   const form = useForm<SendFormValues>({
     resolver,
-    mode: 'onSubmit',
+    mode: 'all',
     defaultValues: DEFAULT_VALUES,
     context: {
       balances: account?.balances,
       baseFee,
       gasLimit,
       maxGasLimit,
     },
+    delayError: 1000,
   });
```

Comment:
we can revert this change to reduce impact as the useForm is a bit chaotic


we should implement this validator on the service, only when the user clicks on "send"